.pre-commit-config.yaml

```diff
@@ -32,7 +32,7 @@ repos:
       - id: fix-byte-order-marker
       - id: name-tests-test
         args: [--pytest-test-first]
-
+        exclude: ^app/core/tests/utils/
```

Comment:
It looks like files currently don't exist in this directory, but files will be added in PR #401. How do we decide that these files are excluded? I have no issue with this ignore, but I'm not sure about the reason. Don't we want it to check those files too?